testutils/gke/client.go

```diff
@@ -109,12 +110,23 @@ func (gsc *sdkClient) GetCluster(project, region, zone, clusterName string) (*co
 	return gsc.Projects.Locations.Clusters.Get(clusterFullPath).Context(context.Background()).Do()
 }
 
+// ListClustersInProject lists all the GKE clusters created in the given project.
+func (gsc *sdkClient) ListClustersInProject(project string) ([]*container.Cluster, error) {
+	var clusters []*container.Cluster
+	projectPath := fmt.Sprintf("projects/%s", project)
+	resp, err := gsc.Projects.Locations.Clusters.List(projectPath).Do()
+	if err != nil {
+		return clusters, fmt.Errorf("failed to list clusters under project %s: %v", project, err)
+	}
+	return resp.Clusters, nil
+}
+
 // GetOperation gets the operation ref with the given operation name.
 func (gsc *sdkClient) GetOperation(project, region, zone, opName string) (*container.Operation, error) {
 	location := GetClusterLocation(region, zone)
 	if zone != "" {
 		return gsc.Service.Projects.Zones.Operations.Get(project, location, opName).Do()
 	}
-	name := fmt.Sprintf("projects/%s/locations/%s/operations/%s", project, location, opName)
-	return gsc.Service.Projects.Locations.Operations.Get(name).Do()
+	clusterFullPath := fmt.Sprintf("projects/%s/locations/%s/operations/%s", project, location, opName)
```

Comment:
this should be named `opsFullPath` to be accurate

Reply:
Done.